Improve documentation about environment variables being used (#138)
kubernetes-sigs/kube-scheduler-simulator: simulator/docs/env-variables.md

# Different Environment Variables used

This page describes about the different environment variables that are
used to configure kube-scheduler-simulator. 

Please refer [docker-compose.yml](./docker-compose.yml) as an example
use.

> `Port`: (required) This is the port number on which kube-scheduler-simulator

Reviewer: ```suggestion
`PORT`: (required) This is the port number on which kube-scheduler-simulator
```

server is started.

`KUBE_SCHEDULER_SIMULATOR_ETCD_URL`: (required) This is the URL for
etcd. The simulator runs kube-apiserver internally, and the
kube-apiserver uses this etcd.

> `FrontendURL`: This URL represents the URL web UI started. The

Reviewer: It's already renamed to CORS_ALLOWED_ORIGIN_LIST.

Author: Updated.

simulator and internal kube-apiserver set the origin as the allowed
origin for `CORS_ALLOWED_ORIGIN_LIST`.

`KUBECONFIG`: This is for the beta feature "Existing cluster
Importing". This variable is used to find Kubeconfig required to
access your cluster for importing resources to scheduler simulator.

> `KUBE_API_HOST`: This is the host of kube-apiserver which the
> simulator starts internally.

Reviewer: ```suggestion
`KUBE_API_HOST`: This is the host of kube-apiserver which the
simulator starts internally. Its default value is `3131`.
```

Reviewer: And `KUBE_API_PORT` env should be mentioned as well. `KUBE_API_PORT` is the port of kube-apiserver.

Author: Updated.

`KUBE_SCHEDULER_CONFIG_PATH`: The path to a KubeSchedulerConfiguration
file.  If passed, the simulator will start the scheduler with that
configuration.  Or, if you use web UI, you can change the
configuration from the web UI as well.

> `ExternalImportEnabled`: This variable indicates whether the simulator will import resources from an existing cluster or not.

Reviewer: ```suggestion
`ExternalImportEnabled`: This variable indicates whether the simulator will import resources from an existing cluster or not. Note that this is still a beta feature.
```

Author: Updated.

## For Web UI

> 

Reviewer: Also, for web ui, ALPHA_TABLE_VIEWS and BASE_URL should be mentioned.
They do the followings. I am not a good English speaker, feel free to change the wording as you like.

BASE_URL → the URL for the simulated server.
ALPHA_TABLE_VIEWS → The value is 0 or 1. (0 means disabled, 1 means enabled.) Enable the alpha feature "table view". You can see the resource status in the table.

Author: Updated as suggested above.

# Different Environment Variables used

This page describes about the different environment variables that are
used to configure kube-scheduler-simulator. 

Please refer [docker-compose.yml](./docker-compose.yml) as an example
use.

`PORT`: (required) This is the port number on which kube-scheduler-simulator
server is started.

`KUBE_SCHEDULER_SIMULATOR_ETCD_URL`: (required) This is the URL for
etcd. The simulator runs kube-apiserver internally, and the
kube-apiserver uses this etcd.

`CORS_ALLOWED_ORIGIN_LIST`: This URL represents the URL once web UI is
started. The simulator and internal kube-apiserver set the allowed
origin for `CORS_ALLOWED_ORIGIN_LIST`.

`KUBECONFIG`: This is for the beta feature "Existing cluster
Importing". This variable is used to find Kubeconfig required to
access your cluster for importing resources to scheduler simulator.

> `KUBE_API_HOST`: This is the host of kube-apiserver which the
> simulator starts internally. It's default value is `3131`.
> 
> `KUBE_API_PORT`: This is the port of kube-apiserver.

Reviewer: Sorry, what I suggested before was wrong 😓 

```suggestion
`KUBE_API_HOST`: This is the host of kube-apiserver which the
simulator starts internally. It's default value is `127.0.0.1`.

`KUBE_API_PORT`: This is the port of kube-apiserver.  It's default value is `3131`.
```

Author: Done. Updated with latest push.

# Different Environment Variables used

This page describes about the different environment variables that are
used to configure kube-scheduler-simulator. 

> Please refer [docker-compose.yml](./docker-compose.yml) as an example

Reviewer 2: ```suggestion
Please refer [docker-compose.yml](./../../docker-compose.yml) as an example
```
This link maybe not working.

Author: Updated.

use.

`PORT`: (required) This is the port number on which kube-scheduler-simulator
server is started.

`KUBE_SCHEDULER_SIMULATOR_ETCD_URL`: (required) This is the URL for
etcd. The simulator runs kube-apiserver internally, and the
kube-apiserver uses this etcd.

`CORS_ALLOWED_ORIGIN_LIST`: This URL represents the URL once web UI is
started. The simulator and internal kube-apiserver set the allowed
origin for `CORS_ALLOWED_ORIGIN_LIST`.

`KUBECONFIG`: This is for the beta feature "Existing cluster
Importing". This variable is used to find Kubeconfig required to
access your cluster for importing resources to scheduler simulator.

`KUBE_API_HOST`: This is the host of kube-apiserver which the

> simulator starts internally. It's default value is `127.0.0.1`.

Reviewer: ```suggestion
simulator starts internally. Its default value is `127.0.0.1`.
```

Author: Updated.

`KUBE_API_PORT`: This is the port of kube-apiserver. It's default
value is `3131`.

`KUBE_SCHEDULER_CONFIG_PATH`: The path to a KubeSchedulerConfiguration
file.  If passed, the simulator will start the scheduler with that
configuration.  Or, if you use web UI, you can change the
configuration from the web UI as well.

`ExternalImportEnabled`: This variable indicates whether the simulator
will import resources from an existing cluster or not. Note, this is
still a beta feature.

## For Web UI

> `KUBE_API_SERVER_URL`: This is the kube-apiserver URL and it's value is set [here](https://github.com/kubernetes-sigs/kube-scheduler-simulator/blob/master/docker-compose.yml#L26)
> 
> `BASE_URL`: This is the URL for the kube-scheduler-simulator server.
> 
> `ALPHA_TABLE_VIEWS`: This variable enables the alpha feature `table
> view`. It's value is either 0 or 1 (0 meaning disalbed, 1 meaning
> enabled). We can see the resource status in the table.

Reviewer 2: ```suggestion
`KUBE_API_SERVER_URL`: This is the kube-apiserver URL and it's value is set [here](https://github.com/kubernetes-sigs/kube-scheduler-simulator/blob/master/docker-compose.yml#L26). Its default value is `http://localhost:3131`.

`BASE_URL`: This is the URL for the kube-scheduler-simulator server. Its default value is `http://localhost:1212`.

`ALPHA_TABLE_VIEWS`: This variable enables the alpha feature `table
view`. It's value is either 0 or 1 (0 meaning disabled, 1 meaning
enabled). We can see the resource status in the table. Its default value is `0`.
```

Reviewer: And please remove `it's value is set [here](https://github.com/kubernetes-sigs/kube-scheduler-simulator/blob/master/docker-compose.yml#L26).`

Author: Updated.
